frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -270,6 +278,14 @@ history-Translation--button-not-rejected =
 history-Translation--button-rejected =
     .title = Rejected
 
+history-Translation--button-comment =
+    .title = Toggle translation comment
```

Comment:
The attribute should come after the value.

In order to easily verify that a string is effectively localized, you can turn on [pseudo-localization](https://github.com/mozilla/pontoon/tree/master/frontend#pseudo-localization).

Reply:
I actually meant to write a comment about this section. I did test it with the pseudolocalization, and it wasn't working (even after a rebuild). 

I just made the changes you stated here, rebuilt, and it still is not showing that element as localized, neither the button text or the title.

Reply:
Hold that thought....I just saw that when I stashed some changes I had in order to run the build that change was stashed as well.

Reply:
Nope...It still doesn't appear to be working.

Reply:
This still did not work.  I just realized though that none of the elements I localized in this section are showing as localized with the pseudolocalizer active so maybe its something with my local environment??

Reply:
Did you `make build` before testing? I'm not sure this is documented anywhere, but it's a requirement before testing localizations.

Reply:
Yes, I ran make build but it still doesn't show as localized.

Reply:
It must have been a caching issue because it is all working fine now (I did clear my cache before, or at least I thought I did 😄 )